nettacker/database/db.py

```
@@ -379,10 +818,30 @@ def get_options_by_scan_id(scan_id):
+                (scan_id,),
+            )
+            rows = cursor.fetchall()
+            if rows:
+                return [{"options": row[0]} for row in rows]
+            return []
+        finally:
+            try:
+                cursor.close()
+                connection.close()
+            except Exception:
+                pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)